backend/api/views/factories_cr.py

```diff
@@ -140,9 +140,9 @@ def _handle_create_ohshown_events(request):
     }
 
     new_reporter_field = {
-        "contact_name": post_body["contactName"],
-        "contact_phone": post_body["contactPhone"],
-        "contact_mail": post_body["contactMail"],
+        "contact_name": post_body.get("contactName"),
```

Comment:
For optional argument, we may consider to use try/except statement to be more "pythonic". See this essay https://devblogs.microsoft.com/python/idiomatic-python-eafp-versus-lbyl/ .